tests/e2e/conftest.py

```diff
@@ -52,26 +53,29 @@ def e2e_test_secrets(request: FixtureRequest) -> EndToEndTestSecrets:
     raise ValueError(f"Unknown e2e_env: {e2e_env}.")
 
 
+@pytest.fixture
+def email(request: FixtureRequest) -> str:
+    email_env = os.getenv("E2E_GOVUK_NOTIFY_AUTHENTICATE_AS")
```

Comment:
```suggestion
    email_env = os.getenv("E2E_AUTHENTICATE_AS")
```

probably nice to keep the env var and the option in sync name-wise.

I'm also a bit 🤷‍♂️  on introducing env vars to toggle these things, esp if we're doing it only partially, but not going to block.

Reply:
Also, this option strikes me more as one that might be configured per-test rather than per-session? Esp once we have multiple user types. This was supported previously with the user_auth fixture and the `user_domain` marker that we're now ripping out. I guess it's fine in the short term but I feel like I'll need to add it back in again soon 🤷

Reply:
👍 yeah agree with all of this, based on your other thoughts I think we'll just remove it in favour of a mark and a default value which would encourage tests to override it if needed

I think it will work fine to be able to have different auth'd sessions per test (and should all work nicely with keeping one session per run if that fixture uses the mark as well) but I think we can keep an eye on and hopefully keep end to end tests to stick to as few users as possible and really cover user types & permissions in detail in integration tests